src/checkErrors.js

```diff
@@ -0,0 +1,34 @@
+const checkErrors = (text, toCase) => {
+  const caseNames = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const errors = [];
+
+  if (!text) {
+    errors.push({
+      message:
+        'Text to convert is required.'
+        + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
This repeated part of the message ("Correct request ...") can be moved to a `const`